doc/contributing/development-environment.md

```diff
@@ -144,9 +144,7 @@ Run any of these tasks with `yarn gulp <task name>` from within the TerriaJS dir
 - `release` - The same as `build` except that it also minifies the build tests.
 - `lint` - Runs ESLint on the files in the `lib` folder and reports any problems. The ESLint rules are defined in the `.eslintrc.js` file in the root directory of TerriaJS.
 - `docs` - Generates the user guide and reference documentation. The user guide is served at `http://localhost:3002/doc/guide/` and the reference documentation is at `http://localhost:3002/doc/reference/`.
-- `make-schema` - Generates [JSON Schema](http://json-schema.org/) for the TerriaJS [Initialization Files](../customizing/initialization-files.md) from the source code. The schema is written to `wwwroot/schema`.
 - `test` - Detects browsers available on the local system and launches the test suite in each. The results are reported on the command line.
-- `test-electron` - Runs the tests in Electron, a headless (no UI) Chrome-like browser.
```

Comment:
This is replaced with `test-firefox` which is not documented anywhere, it might be better to change this to 
```
`test-firefox` - Runs tests in headless Firefox browser.
```

Reply:
Fixed.